pykechain/client.py

```diff
@@ -1177,17 +1177,17 @@ def clone_activities(
         activities = [dict(id=uuid, **update_dicts.get(uuid, {})) for uuid in activity_ids]
```

Comment:
om line 1161 we need to change the version of the `match_app_version` to `version=">='3.7.0'"`

I changed the pim version of the backend to support the clone activity with parts in version 3.7.0 of the pim django 'app'.